packages/utils/package.json

```diff
@@ -8,7 +8,7 @@
     "xmldoc": "^1.1.2",
     "tslib": "^2.5.0",
     "semver": "7.6.3",
-    "@nx/devkit": "21.0.3"
+    "@nx/devkit": "^20.0.0 || ^21.0.0 || ^22.0.0"
```

Comment:
This isn't correct, the plugin should have a dependency on @nx/devkit@21, which supports Nx 20 -> 22. There's a bug in devkit that we just patched out where that dependency range got messed up.

After that gets released we'll bump the plugin to the proper devkit dep. ref: https://github.com/nrwl/nx/pull/32406